src/script/script_info.cpp

```diff
@@ -232,8 +232,8 @@ SQInteger ScriptInfo::AddLabels(HSQUIRRELVM vm)
 	ValidateString(setting_name);
 
 	ScriptConfigItem *config = nullptr;
-	for (ScriptConfigItemList::iterator it = this->config_list.begin(); it != this->config_list.end(); it++) {
-		if (strcmp((*it).name, setting_name) == 0) config = &(*it);
+	for (auto &item : this->config_list) {
```

Comment:
this one can be const

Reply:
Compiler says no:

`script_info.cpp:236:54: error: invalid conversion from ‘const ScriptConfigItem*’ to ‘ScriptConfigItem*’ [-fpermissive]
  236 |   if (strcmp(item.name, setting_name) == 0) config = &item;`

(I tried `const` first for all, if fail, removed `const`)

Reply:
Ah yes, because `config` is not const